lib/resources/security_policy.rb

```diff
@@ -108,22 +109,25 @@ def to_s
     def read_content
       return @content if defined?(@content)
 
+      # using random hex to prevent any race conditions with multiple runners
+      export_file = "win_secpol-#{SecureRandom.hex}.cfg"
+
       # export the security policy
-      cmd = inspec.command('secedit /export /cfg win_secpol.cfg')
+      cmd = inspec.command("secedit /export /cfg #{export_file}")
       return nil if cmd.exit_status.to_i != 0
 
       # store file content
-      cmd = inspec.command('Get-Content win_secpol.cfg')
+      cmd = inspec.command("Get-Content #{export_file}")
       return skip_resource "Can't read security policy" if cmd.exit_status.to_i != 0
       @content = cmd.stdout
 
-      if @content.empty? && !file.empty?
```

Comment:
This was removed as it seems to be legacy code. We don't have any local `file` variable. In the future we could look into using the Train::File to capture the content of the policy.

Reply:
I think we should remove the content check completely. If the content is empty, we should not expect that to be okay, so it is probably a failure. Can we add a test that ensures the parsing is not crashing if we have an empty file?